spec/mongo/auth/scram/conversation_spec.rb

```diff
@@ -1,12 +1,13 @@
 require 'spec_helper'
 
-describe Mongo::Auth::SCRAM::Conversation do
+describe Mongo::Auth::SCRAM::Conversation, if: scram_sha_1_enabled? do
 
   let(:user) do
     Mongo::Auth::User.new(
       database: Mongo::Database::ADMIN,
       user: 'user',
-      password: 'pencil'
+      password: 'pencil',
+      auth_mech: 'SCRAM-SHA-1'
```

Comment:
SCRAM-SHA-1 is explicitly set here and elsewhere that we're explicitly testing it so we don't accidentally use SCRAM-SHA-256 with newer server versions.

Reply:
This one